Buy/Client/Graph.Client.swift

```diff
@@ -79,8 +79,9 @@ extension Graph {
         ///     - shopDomain: The domain of your shop (ex: "shopname.myshopify.com").
         ///     - apiKey:     The API key for you app, obtained from the Shopify admin.
         ///     - session:    A `URLSession` to use for this client. If left blank, a session with a `default` configuration will be created.
+        ///     - locale:    The buyer's locale
         ///
-        public init(shopDomain: String, apiKey: String, session: URLSession = URLSession(configuration: URLSessionConfiguration.default)) {
+        public init(shopDomain: String, apiKey: String, session: URLSession = URLSession(configuration: URLSessionConfiguration.default), locale: Graph.Locale = Graph.Locale.null) {
```

Comment:
Any nested types don't need to be explicitly referenced. We can simplify to this:
```swift
... locale: Locale? = nil) {
```
Side note: Optionality is built into Swift (see comment below). Having said that, I think we should default to `.en` instead of `nil`.

Reply:
Thanks will fix nullable locale param

> Side note: Optionality is built into Swift (see comment below). Having said that, I think we should default to .en instead of nil.

When users of the sdk pass in a value that is not supported, we default not to the en  but the shop langauge